GOVERNANCE.md

```diff
@@ -110,3 +171,9 @@ Lazy consensus does _not_ apply to the process of:
 ## Updating Governance
 
 All substantive changes in Governance require a supermajority agreement by all maintainers.
+
+
+## Thanks
```

Comment:
Do you want to add in acknowledgements on where some of the new content came from?
Something like "some content in this document were built upon the work in the Kubernetes community" with a link? If that's true. I'm not 100% sure where it comes from :)